docs/standard/basetypes/alternation.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Alternation constructs in regular expressions
+description: Alternation constructs in regular expressions
+keywords: .NET, .NET Core
+author: stevehoag
+manager: wpickett
+ms.date: 07/28/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 59ffac4d-fc6e-461f-8783-d9f8dc88ce2c
```

Comment:
should we use the AssetID from msdn on these topics instead? (I really don't know the answer to this question)